### PR TITLE
feat(section/info): adds a new section info component

### PR DESCRIPTION
**Pull request by davecarter**
### Adds a new section component:

screen shot 2017-11-30 at 11 32 22

### CODE REVIEW

File: components/section/info/src/index.scss [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+.sui-SectionInfo {
+  @include media-breakpoint-down(m) {
+    flex-direction: column;
+  }
+  
+  border-bottom: 1px solid $c-sui-section-info-border;
+  display: flex;
+  flex-direction: row;
+  max-width: 670px;
```
Comment (kikoruiz) [inline components/section/info/src/index.scss]: Magic number for everyone?

File: components/section/info/src/index.scss [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  padding: $p-xxl 0;
+
+  &-title {
+    @include media-breakpoint-down(m) {
+      flex-basis: auto;
+      padding: $p-l 0;
+      text-align: center;
+    }
+    
+    flex-basis: 30%;
```
Comment (kikoruiz) [inline components/section/info/src/index.scss]: This value and the one from line [33](https://github.com/SUI-Components/sui-components/blob/80da9f77393bf9121051974e9435a06cadf141a6/components/section/info/src/index.scss#L33) could be saved as variables as well.

File: components/section/info/src/index.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+import PropTypes from 'prop-types'
+
+const baseClass = 'sui-SectionInfo'
+
+class SectionInfo extends Component {
+  render () {
+    const { title, children } = this.props
+
+    return (
+      <section className={`${baseClass}`}>
```
Comment (carlesnunez) [inline components/section/info/src/index.js]: Whats the point of run over interpolation here?

File: components/section/info/src/index.scss [hunk 4]
```diff
@@ -0,0 +1,38 @@
+@import '~@schibstedspain/theme-basic/lib/index';
+@import './settings';
+
+.sui-SectionInfo {
+  @include media-breakpoint-down(m) {
+    flex-direction: column;
+  }
+  
+  border-bottom: 1px solid $c-sui-section-info-border;
```
Comment (carlesnunez) [inline components/section/info/src/index.scss]: We should be EMMET compliant and declare some of our props as EMMET variables:
`$bdb-section-info: 1px solid $c-sui-section-info-border`

https://docs.emmet.io/cheat-sheet/

File: components/section/info/README.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
+# SectionInfo
```
Comment (carlesnunez) [inline components/section/info/README.md:1]: Should we provide further info about the usage of the component?

File: components/section/info/README.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+$ npm install @schibstedspain/sui-section-info --save
+```
+
+## Usage
+
+### Basic usage
+```js
+import SectionInfo from '@schibstedspain/sui-section-info'
+
+return (<SectionInfo />)
```
Comment (midudev) [inline components/section/info/README.md]: Missing good example.

File: components/section/info/README.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
+# SectionInfo
+
+> Description
```
Comment (midudev) [inline components/section/info/README.md]: A small description would be welcome!

File: components/section/info/src/index.scss [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+    margin: 0;
+    padding: $p-l $p-m;
+  }
+
+  &-content {
+    @include media-breakpoint-down(m) {
+      flex-basis: auto;
+    }
+
+    flex-basis: 70%;
```
Comment (midudev) [inline components/section/info/src/index.scss]: Potential value to store in a variable.

File: demo/section/info/playground [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+const description = 'Description'
+
+const extrasElement = <ul><li>air conditioning</li><li>lifter</li><li>terace</li><li>pool</li><li>parking</li><li>solar powered</li><li>kitchen</li><li>toilett</li><li>garden</li><li>appliances</li></ul>
+const extras = 'Extras'
+
+const featuresElement = <ul><li>Type of property : Chalet</li><li>Type of property : Chalet</li><li>Orientation : Orientation South</li></ul>
+const features = 'Features'
+
+return (
+  <fragments>
```
Comment (midudev) [inline demo/section/info/playground]: Use `<div>` instead, as `<Fragment>` is not available in the playground.

**Review by midudev — APPROVED** [review]

**Review by davidbarna — APPROVED** [review]

**Pull request merged**